src/ethers/signer.ts

```diff
@@ -199,3 +291,77 @@ export class KlaytnWallet extends Wallet {
     }
   }
 }
+
+export async function verifyMessageAsKlaytnAccountKey(provider: Provider, address: string, message: Bytes | string, signature: any): Promise<boolean> {
```

Comment:
Is there a non-networking function?
e.g. `verifyMessageWithKlaytnAccountKey(message, signature, accountInfo or accountKey)`

Reply:
Klaytn AccountKey might be updated. 
So I think it has to be networking function. 
In case that AccountKey is not updated, non-networking function "ethers.utils.verifyMessage" will be used.